uktrade/data-hub-frontend
# Refactor company export pages to use new template

## Description of change
The company export pages have been refactored to use the new layout introduced in https://github.com/uktrade/data-hub-frontend/pull/5682. I also had to make a small update to the company overview controlller to update the usage of the export countries transformer.

## Test instructions

Go to the exports page/export history page for a company. Everything should work as before.

## Screenshots

See screenshots in https://github.com/uktrade/data-hub-frontend/pull/5682. There are no visual changes to the pages themselves (only that the company tab navigation is now visible on the export history page).

## Checklist

[//]: # 'When submitting a PR make sure the code review guidelines have been satisfied.
https://github.com/uktrade/data-hub-frontend/blob/main/docs/Code%20review%20guidelines.md'
